modules/machine-edge-pool-review-nodes.adoc

```diff
@@ -0,0 +1,56 @@
+
+:_content-type: PROCEDURE
+[id="machine-edge-pool-review-nodes_{context}"]
+= Verifying nodes that were created with edge compute pool
+
+After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
```

Comment:
Change to: "After you install a cluster..."

Reply:
```suggestion
After you install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
```